packages/mui-styled-engine-sc/src/index.d.ts

```diff
@@ -21,6 +21,11 @@ export { default as StyledEngineProvider } from './StyledEngineProvider';
 export { default as GlobalStyles } from './GlobalStyles';
 export * from './GlobalStyles';
 
+/**
+ * For internal usage in `@mui/system` package
+ */
+export function processStyles(tag: React.ElementType, processor: (styles: any) => any): void;
```

Comment:
Can we add some prefix (`internal_` or `unstable_`)?

Reply:
good point.